include/drivers/watchdog.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef DRIVERS_WATCHDOG_H
+#define DRIVERS_WATCHDOG_H
+
+#include <dm.h>
+#include <stdint.h>
+
+#define WATCHDOG_OPS(dev) ((struct watchdog_driver_ops *)((dev)->drv->ops))
+
+struct watchdog_driver_ops {
+	int (*disable)(struct device *dev);
+	int (*enable)(struct device *dev, uint32_t timeout);
+};
+
+static inline int
+watchdog_disable(struct device *dev)
```

Comment:
This file is good, but the functions need comments.

Reply:
Done.